lib/dry/schema/extensions/hints/message_set_methods.rb

```diff
@@ -61,7 +62,7 @@ def messages_map(messages = self.messages)
           # rubocop:disable Metrics/AbcSize
           # rubocop:disable Metrics/PerceivedComplexity
           def initialize_placeholders!
-            @placeholders = unique_paths.each_with_object(EMPTY_HASH.dup) { |path, hash|
+            @placeholders ||= unique_paths.each_with_object(EMPTY_HASH.dup) { |path, hash|
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Memoized variable `@placeholders` does not match method name `initialize_placeholders!`. Use `@initialize_placeholders` instead.](https://app.codacy.com/gh/dry-rb/dry-schema/pullRequest?prid=4764391)